packages/mobile/scripts/run_e2e.sh

```diff
@@ -190,12 +189,6 @@ if [ $PLATFORM = "android" ]; then
   # Run Detox Tests
   runTest
 
-  # Close active emulators 
```

Comment:
is detox closing the emulators after the tests by default? or are we leaving the emulators running until the next run?

Reply:
Added this back in, as it does appear as if it was closing the emulators in CI after the run.